src/main/java/org/sasanlabs/service/vulnerability/xss/reflected/XSSWithHtmlTagInjection.java

```diff
@@ -85,4 +85,70 @@ public ResponseEntity<String> getVulnerablePayloadLevel3(
         }
         return new ResponseEntity<String>(payload.toString(), HttpStatus.OK);
     }
+
+    // Just adding User defined input(Untrusted Data) into div tag if doesn't contains
+    // eval(...) expression which evaluates the string expression and returns its value.
+    // Can be broken by various ways
+
+    @AttackVector(
+            vulnerabilityExposed = VulnerabilityType.REFLECTED_XSS,
+            description = "XSS_DIRECT_INPUT_DIV_TAG_AFTER_REMOVING_VALUES_CONTAINING_EVAL_EXPRESSION")
+    @VulnerableAppRequestMapping(value = LevelConstants.LEVEL_8, htmlTemplate = "LEVEL_1/XSS")
+    public ResponseEntity<String> getVulnerablePayloadLevel8(
+            @RequestParam Map<String, String> queryParams) {
+        String vulnerablePayloadWithPlaceHolder = "<div>%s<div>";
+        StringBuilder payload = new StringBuilder();
+        Pattern pattern = Pattern.compile("eval\\((.*?)\\)");
+        for (Map.Entry<String, String> map : queryParams.entrySet()) {
+            Matcher matcher = pattern.matcher(map.getValue());
+            if (!matcher.find()) {
+                payload.append(String.format(vulnerablePayloadWithPlaceHolder, map.getValue()));
+            }
+        }
+        return new ResponseEntity<String>(payload.toString(), HttpStatus.OK);
+    }
+
+    // Just adding User defined input(Untrusted Data) into div tag if doesn't contains
+    // Paranoid regex.
+    // Can be broken by various ways
+
+    @AttackVector(
+            vulnerabilityExposed = VulnerabilityType.REFLECTED_XSS,
+            description = "XSS_DIRECT_INPUT_DIV_TAG_AFTER_REMOVING_VALUES_CONTAINING_PARANOID")
+    @VulnerableAppRequestMapping(value = LevelConstants.LEVEL_9, htmlTemplate = "LEVEL_1/XSS")
+    public ResponseEntity<String> getVulnerablePayloadLevel9(
+            @RequestParam Map<String, String> queryParams) {
+        String vulnerablePayloadWithPlaceHolder = "<div>%s<div>";
+        StringBuilder payload = new StringBuilder();
+        Pattern pattern = Pattern.compile("<script(.*?)[\r\n]*(.*?)/script>");
+        for (Map.Entry<String, String> map : queryParams.entrySet()) {
+            Matcher matcher = pattern.matcher(map.getValue());
+            if (!matcher.find()) {
+                payload.append(String.format(vulnerablePayloadWithPlaceHolder, map.getValue()));
+            }
+        }
+        return new ResponseEntity<String>(payload.toString(), HttpStatus.OK);
+    }
+
+    // Just adding User defined input(Untrusted Data) into div tag if contains
+    // onLoad expression which deals with the cookies.
+    // Can be broken by various ways
+    @AttackVector(
+            vulnerabilityExposed = VulnerabilityType.REFLECTED_XSS,
+            description =
+                    "XSS_DIRECT_INPUT_DIV_TAG_AFTER_REMOVING_VALUES_CONTAINING_ONLOAD_EXPRESSION")
+    @VulnerableAppRequestMapping(value = LevelConstants.LEVEL_10, htmlTemplate = "LEVEL_1/XSS")
```

Comment:
so we can reformat, level 8 method which you created can be called level 2, level 9 can be called level 3, level 10 can be called level 4 and then currently level 2 and level 3 can be called level 5 and level 6 respectively.